relay_handler.js

```diff
@@ -63,10 +63,14 @@ RelayHandler.prototype.handleLazily = function handleLazily(conn, reqFrame) {
     }
 
     if (self.circuits) {
-        self.logger.warn(
-            'circuit breaking for lazy relaying isn\'t implemented',
-            rereq.extendLogInfo({}));
-        return false;
+        var circuit = self.circuits.getCircuit(rereq.callerName, rereq.serviceName, rereq.endpoint);
```

Comment:
I just want consistency so its easy to manually verify that eager and lazy are the same.

Let's move the circuits code out of relay_handler into service-proxy (real soon tm)